{
  "_annotation": {
    "cve_id": "CVE-2024-3116",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-3116.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:pgadmin:pgadmin:*:*:*:*:*:postgresql:*:*",
                "matchCriteriaId": "88F55FF8-B13F-5B21-865A-26339C0A1EAC",
                "versionEndExcluding": "8.5",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          },
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:pgadmin:pgadmin_4:*:*:*:*:*:postgresql:*:*",
                "matchCriteriaId": "B6B41BA4-FF55-5C74-92F8-2BBA7DBA5E30",
                "versionEndExcluding": "8.5",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}